src/data.tsx

```diff
@@ -0,0 +1,94 @@
+      "source": "http://www.trulia.com"
+    },
+    "credit_score_range":{
+      "type": "Credit Score", 
+      "question": "What is your current credit score?",
+      "description": " A high score will give you access to lower interest rates and more lender choices. If you have a low score, you may have trouble getting a loan.",
+      "information": "Your credit score plays a big role in the interest rate you’ll get for your loan.",
+      "note": "Your credit score is a numerical rating that ranges from 300 – 850 and tells lenders how responsible you are when you borrow money. ",
+      "source": "https://www.rocketmortgage.com"
+    },
+    "downpayment_percentage": {
+      "type": "Downpayment Percentage", 
+      "question": "What downpayment percentage are you aiming to save?",
+      "definition": "Your down payment is the amount of money you put down on your mortgage. Your down payment is due during closing and is usually the most expensive closing cost you need to plan for.",
+      "information": "You might have heard you need 20% down to buy a home. The reason why this number is often quoted is that 20% down is the minimum you’ll need to avoid buying private mortgage insurance – it’s not the minimum you need to get a loan.",
+      "note":"You can buy a home with as little as 3% down.",
+      "source": "https://www.rocketmortgage.com"
+    },
+    "mortgage_term": {
+      "type": "Mortgage Term", 
+      "question": "What mortgage term are you aiming for?",
+      "description": "Your mortgage term is the number of years you’ll pay on your loan before you fully own your home. ",
+      "information": "A mortgage is a type of loan that uses your home as collateral. It’s typically used to buy a home or in refinance situations to secure a more favorable deal and possibly convert equity in your home to cash.",
+      "note": "Typically, a fixed-rate mortgage runs 30 years and this is the number we will use if you want us to.",
+      "source": "https://www.rocketmortgage.com/"
+    },
+    "rent": {
+      "type": "Rent", 
+      "question": "IMAGINATIVE: What is your current monthly rent?",
+      "description": "",
+      "information": "",
+      "note": "The imaginative report gives a total home price you can afford based off of your monthly rent.",
+      "source": ""
+    },
+    "goal_home_price": {
+      "type": "Goal Home Price", 
+      "question": "PRAGMATIC: What is the price of your ideal home?",
+      "description": "",
+      "information": "",
+      "note": "The pragmatic report gives your monthly payment you would make with your goal price.",
+      "source": ""
+    },
+    "external_services": {
+      "type": "",
+      "property_tax": "number",
+      "description": "",
+      "information": "",
+      "note":"",
+      "affordability": "mortgage amount + average mortgage from zipcode"
+    },
```

Comment:
We can get rid of this section, it goes into a different model we're calling the Report Model.